back/spec/models/event_spec.rb

```diff
@@ -9,6 +9,16 @@
     end
   end
 
+  describe 'associations' do
+    subject(:event) { build(:event) }
+
+    it { is_expected.to belong_to(:project) }
+    it { is_expected.to have_many(:attendances).dependent(:destroy) }
+    it { is_expected.to have_many(:attendees).through(:attendances) }
+    it { is_expected.to have_many(:event_files).dependent(:destroy) }
+    it { is_expected.to have_many(:text_images).dependent(:destroy) }
```

Comment:
Kudos to the extensive coverage, but not too convinced whether these are worth the maintenance time.

Reply:
I think they are pretty standard and low cost, including in terms of maintenance. I mean, it took me two minutes to implement. (I'm lying: 3 minutes.)